docs/docs/reference/built-in-components/gatsby-plugin-image.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Gatsby Image plugin
+---
+
+This guide will show you how to configure your images, including choosing layouts, placeholders and image processing options. If you would like to learn how to set up the image plugins and use images on your site, see [the how-to guide](/docs/how-to/images-and-media/using-gatsby-plugin-image).
```

Comment:
> If you would like to learn how to set up the image plugins and use images on your site, see [the how-to guide](/docs/how-to/images-and-media/using-gatsby-plugin-image).

I'd move this sentence down after you list out the components. So that it's like "Here are the two components you can use. If you want an example, go to the how-to guide."

Reply:
Maybe in this introduction we could put a similar callout as in the other doc, sending people to the Conceptal Guide about image optimizations.